tools/windows/install-help/cal/userrights.cpp

```diff
@@ -328,7 +328,7 @@ DWORD AddUserToGroup(PSID userSid, wchar_t *groupSidString, wchar_t *defaultGrou
     nErr = NetLocalGroupAddMembers(NULL, groupname.c_str(), 0, (LPBYTE)&lmi0, 1);
     if (nErr == NERR_Success)
     {
-        WcaLog(LOGMSG_STANDARD, "Added ddagentuser to %s", asciiname.c_str());
+        WcaLog(LOGMSG_STANDARD, "Added ddagentuser to %S", asciiname.c_str());
```

Comment:
same.
And, `asciiname` isn't filled in anywhere, so it's always blank.
Presumably a long time ago there was a wide->ascii conversion (the only reason for the `asciiname` varible is for logging), which was lost.
So for this one, could just use `groupname` with `%S`, and get rid of the unused variable.